src/ray/core_worker/core_worker.cc

```diff
@@ -635,6 +624,14 @@ std::string CoreWorker::MemoryUsageString() {
   return plasma_store_provider_->MemoryUsageString();
 }
 
+TaskID CoreWorker::GetInitialCallerId() {
```

Comment:
@stephanie-wang this effectively makes the caller id per-worker for purposes of actor sequencing. This is now needed since the actor handles are shared across multiple tasks.